cyclonedx/output/xml.py

```diff
@@ -30,24 +31,49 @@ class Xml(BaseOutput, BaseSchemaVersion):
     def get_target_namespace(self) -> str:
         return 'http://cyclonedx.org/schema/bom/{}'.format(self.get_schema_version())
 
+    @staticmethod
+    def get_vulnerabilities_namespace() -> str:
+        return 'http://cyclonedx.org/schema/ext/vulnerability/1.0'
+
     def output_as_string(self) -> str:
         bom = self._get_bom_root_element()
 
         if self.bom_supports_metadata():
             bom = self._add_metadata(bom=bom)
 
+        if self.get_bom().has_vulnerabilities():
+            ElementTree.register_namespace('v', Xml.get_vulnerabilities_namespace())
+
         components = ElementTree.SubElement(bom, 'components')
+        # if self.get_bom().has_vulnerabilities():
```

Comment:
purpose of this comment?

Reply:
Good catch @jkowalleck  - should not be there - will fix.